modules/local/alakazam/alakazam_lineage.nf

```diff
@@ -14,14 +14,14 @@ process ALAKAZAM_LINEAGE {
     tuple val(meta), path("${tab}"), emit: tab
     path "versions.yml" , emit: versions
     path("*.tsv")
-    path("Clone_tree_plots/*.pdf")
-    path("Graphml_trees/All_graphs_patient.graphml")
+    path("Clone_tree_plots/*.pdf"), emit: graph_plots optional true
+    path("Graphml_trees/*.graphml"), emit: graph_export optional true
```

Comment:
```suggestion
    path("Clone_tree_plots/*.pdf"), emit: graph_plots, optional true
    path("Graphml_trees/*.graphml"), emit: graph_export, optional true
```

Reply:
seems like a comma there is not supported, just tested this